release-tools/src/main/java/org/springframework/data/release/cli/IssueTrackerCommands.java

```diff
@@ -36,12 +38,14 @@
 
 /**
  * @author Oliver Gierke
+ * @author Mark Paluch
  */
 @CliComponent
 public class IssueTrackerCommands implements CommandMarker {
 
 	private final PluginRegistry<IssueTracker, Project> tracker;
 	private final JiraConnector jira;
+	private final GitHubIssueConnector gitHub;
```

Comment:
As indicated in the general comment I'd rather make use of the `PluginRegistry` to delegate to the right tracker connector.